src/test/functional/features/other-court-cases.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Other court cases
+
+  Background:
+    Given I login
+    And I've said I'm divorcing my husband
+    When I go to '/other-court-cases'
+    Then the page should include "Other court cases"
+
+  Scenario: Failed to fill in the form
+    Given I clear the form
+    When I click "Continue"
+    Then the page should include "There was a problem"
+
+  Scenario: Successfully completing the form
+    Given I select "No"
+    When I click "Continue"
+    Then the page URL should be "/money-property"
+
+  Scenario: Successfully completing the form
```

Comment:
Reusing the same scenario name generates a warning in the console, perhaps we should use a different scenario name for these two.